eslint.config.mjs

```diff
@@ -0,0 +1,366 @@
+    },
+    {
+        files: ['**/__tests__/**', '**/__mocks__/**', 'packages/@lwc/integration-karma/**'],
+
+        languageOptions: {
+            globals: {
+                ...globals.jest,
+                ...globals.es2021,
+            },
+        },
+
+        plugins: {
+            jest,
+        },
+
+        rules: {
+            'jest/no-focused-tests': 'error',
+            'jest/valid-expect': 'error',
+            'jest/valid-expect-in-promise': 'error',
+        },
+    },
+    {
+        files: ['packages/@lwc/integration-tests/**'],
+
+        languageOptions: {
+            globals: {
+                $: true,
+                browser: true,
+                ...globals.browser,
+                ...globals.mocha,
+                ...globals.node,
+            },
+        },
+    },
+    {
+        files: ['./*.js', '**/scripts/**', '**/jest.config.js'],
+
+        languageOptions: {
+            globals: {
+                ...globals.node,
+                ...globals.jest,
+            },
+        },
+
+        rules: {
+            'no-console': 'off',
+        },
+    },
+    {
+        files: ['packages/@lwc/perf-benchmarks/**'],
+
+        languageOptions: {
+            globals: {
+                after: true,
+                before: true,
+                benchmark: true,
+                run: true,
+                browser: true,
+                ...globals.browser,
+                ...globals.node,
+            },
+        },
+    },
+    {
+        files: ['packages/@lwc/perf-benchmarks-components/**'],
+
+        languageOptions: {
+            globals: {
+                browser: true,
+                ...globals.node,
+            },
+        },
+    },
+    {
+        files: [
+            'packages/@lwc/integration-tests/src/**/!(*.spec.js)',
+            'packages/@lwc/integration-karma/test/**',
+            'packages/@lwc/integration-karma/test-hydration/**',
+        ],
+
+        rules: {
+            'header/header': 'off',
+        },
+    },
+    {
+        files: ['packages/@lwc/integration-karma/**'],
+
+        languageOptions: {
+            globals: {
+                lwcRuntimeFlags: true,
+                process: true,
+                LWC: true,
+                spyOnAllFunctions: true,
+                TestUtils: true,
+                ...globals.browser,
+                ...globals.jasmine,
+            },
+        },
+
+        rules: {
+            'no-var': 'off',
+            'prefer-rest-params': 'off',
+        },
+    },
+    {
+        files: ['packages/@lwc/synthetic-shadow/**'],
+
+        languageOptions: {
+            globals: {
+                process: true,
+                ...globals.browser,
+            },
+        },
+    },
+    {
+        // These are empty files used to help debug test fixtures
+        files: ['**/.only', '**/.skip'],
+        plugins: { '@lwc/lwc-internal': lwcInternal },
+        rules: {
+            '@lwc/lwc-internal/forbidden-filename': 'error',
```

Comment:
Just checking, but I think we used to set this to `'off'` for `.skip` files so that we could commit them if needed. Are we intentionally not supporting that use case now?

Reply:
Has anyone ever actually wanted to commit a skip?

Reply:
You are right, the eslintrc config has it like you described. However, the [lint-staged](https://github.com/salesforce/lwc/blob/master/package.json#L78) section in package.json seems to enforce the rule for both.

I will make the new config match eslintrc

Reply:
@wjhsf We do have `xdescribe` and `xit` in our source ¯\_(ツ)_/¯